### PR TITLE
Fix build issues before pushing circle ci config

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/23238

Splitting of https://github.com/giantswarm/heartbeatctl/pull/3 part 0

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,17 +1,36 @@
...
-	github.com/hashicorp/go-cleanhttp v0.5.1 // indirect
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.20.2
 	github.com/opsgenie/opsgenie-go-sdk-v2 v1.2.13
 	github.com/ryanuber/columnize v2.1.2+incompatible
 	github.com/sirupsen/logrus v1.9.0
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
-	k8s.io/apimachinery v0.20.1
+	k8s.io/apimachinery v0.25.0
```
Comment (QuentinBisson) [inline go.mod:15]: Needed to fix the CVE https://nvd.nist.gov/vuln/detail/cve-2021-3121

File: pkg/client/adapter.go [hunk 2]
```diff
@@ -27,8 +27,9 @@ func New(cfg *client.Config) (Port, error) {
 	}
 
 	if cfg.Logger == nil {
-		cfg.Logger = logrus.New()
-		cfg.Logger.SetLevel(logrus.PanicLevel)
+		logger := logrus.New()
+		logger.SetLevel(logrus.InfoLevel)
```
Comment (QuentinBisson) [inline pkg/client/adapter.go:31]: Need to fix a compilation error after logrus was updated

File: pkg/ctl/adapter_test.go [hunk 3]
```diff
@@ -16,6 +16,12 @@ import (
 	"github.com/giantswarm/heartbeatctl/pkg/mocks"
 )
 
+const (
+	GetMethodName     = "Get"
```
Comment (QuentinBisson) [inline pkg/ctl/adapter_test.go:20]: CI will complain that the string is present too often

**Review by TheoBrigitte — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Not needed

**Pull request closed without merging**